manifests/init.pp

```diff
@@ -6,8 +6,8 @@
 #  class { 'satellite_pe_tools':
 #    satellite_url => 'https://satellite.example.domain',
 #    ssl_ca        => '/etc/puppetlabs/puppet/ssl/ca/katello-default-ca.crt',
-#    ssl_cert      => '/etc/puppetlabs/puppet/ssl/certs/satellite-master.example.domain.pem',
-#    ssl_key       => '/etc/puppetlabs/puppet/ssl/private_keys/master.domain.com.pem',
+#    ssl_cert      => '/etc/puppetlabs/puppet/ssl/certs/satellite-server.example.domain.pem',
+#    ssl_key       => '/etc/puppetlabs/puppet/ssl/private_keys/server.domain.com.pem',
```

Comment:
I still find it odd that the cert filename is `satellite-server.example.domain.pem` (prefixed with `satellite-`, includes `example.`) while `ssl_key` is `server.domain.com.pem`. Normally these always use the `$certname.pem` and should match, right?

Reply:
Have changed this to be inline with the with the earlier change you suggested. Unsure if they should match or not however, as you can probably tell, my knowledge of this module is rather cursory.

Reply:
I must also admit that I've never used this module.